src/mumble/MainWindow.cpp

```diff
@@ -1969,6 +1973,20 @@ void MainWindow::openTextMessageDialog(ClientUser *p) {
 	delete texm;
 }
 
+void MainWindow::on_qaUserLocalNickname_triggered() {
+	ClientUser *p = getContextMenuUser();
+
+	if (!p)
+		return;
+
+	openUserLocalNicknameDialog(p);
+}
+
+void MainWindow::openUserLocalNicknameDialog(ClientUser *p) {
```

Comment:
```suggestion
void MainWindow::openUserLocalNicknameDialog(const ClientUser &p) {
```
`const` as the user shall not be modified by this function and a reference should be used as from the code it is apparent that it will not work if `p == nullptr` and thus it increases the safety of this function, if it also formally forbids this case (by using a reference instead of a pointer)